src/ui/lib/components/build-providence.svelte

```diff
@@ -0,0 +1,51 @@
+<script lang="ts">
+	import { Box, Typography, type SSX, currentTheme } from '@ui';
+	import type { ZarfBuildData } from '$lib/api-types';
+
+	export let build: ZarfBuildData | undefined;
+	const labels = ['terminal', 'user', 'architecture', 'timestamp', 'version'];
```

Comment:
Is there a way to extract the keys from the `ZarfBuildData` interface?

Reply:
I would either need to know which keys I want, or which keys I don't want to ignore the ones we aren't using each solution has the same result on just requires more comparisons.

Reply:
Those keys are also used as labels so I would say by explicitly declaring them is probably the cleaner solution imo than filtering the keys I don't want since that would be introducing explicit strings that are only used to not display something rather than strings that are explicitly relevant.